www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb

```diff
@@ -1,32 +1,32 @@
 # <a name="plan-builds" id="plan-builds" data-magellan-target="plan-builds">Plan Builds</a>
 
-Packages need to be signed with a secret origin key at buildtime. Generate an origin key pair manually by running the following command on your host machine:
+Packages need to be signed with a private origin key at buildtime. Generate an origin key pair manually by running the following command on your host machine:
 
 ```shell
 $ hab origin key generate <ORIGIN>
 ```
 
-The `hab-origin` subcommand will place originname-_timestamp_.sig.key and originname-_timestamp_.pub files (the origin key pair) in the `$HOME/.hab/cache/keys` directory. If you're creating origin keys in the Studio container or you are running as root on a Linux machine, your keys will be stored in `/hab/cache/keys`.
+The `hab-origin` subcommand will place the origin key files, originname-_timestamp_.sig.key (the private key) and originname-_timestamp_.pub files (the public key), in the `$HOME/.hab/cache/keys` directory. If you're creating origin keys in the Studio container, or you are running as root on a Linux machine, your keys will be stored in `/hab/cache/keys`.
 
-Because the secret key is used to sign your package, it should not be shared freely; however, if anyone wants to download and use your package, then they must have your public key (.pub) installed in their local `$HOME/.hab/cache/keys` or `/hab/cache/keys` directory. Public keys will be downloaded from Habitat Builder by the Supervisor, if needed.
+Because the private key is used to sign your package, it should not be shared freely; however, if anyone wants to download and use your package, then they must have your public key (.pub) installed in their local `$HOME/.hab/cache/keys` or `/hab/cache/keys` directory. The Supervisor downloads public keys, if they are needed.
 
 ### Passing origin keys into the Studio
 
-When you enter the Studio environment, your origin keys are not automatically shared into it. This is to keep the Studio environment as clean as possible. However, because you need to reference a secret origin key to sign your package, you can do this in three ways:
+The Habitat Studio is a clean, self-contained, minimal environment, which means that you'll need to share your private origin keys with Studio to sign packages. You can do this in three ways:
```

Comment:
A studio is not necessarily clean; they can be used in a persistent fashion and have things installed. It's easy to get a *new*, pristine studio, but the studio isn't inherently "clean".

Reply:
Studio isn't a "clean-room"?

Reply:
"clean room" describes the purpose, not an inherent quality. If you took a [clean room](https://en.wikipedia.org/wiki/Cleanroom) like is used for industrial production or scientific research and brought a bunch of contaminants into it, it would be misleading to refer to it as "clean", right?

I think it's an important distinction to say that the studio is designed for a clean room approach to building software without unintentional dependencies, but it's definitely possible to render that environment unclean with use.